src/App.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
-import AddTodoForm from './AddTodoForm';
+import React, { useState } from 'react';
+import AddTooForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
+  const [newTodo, setNewTodo] = useState();
```

Comment:
You won't run into an issue right now but not specifying a default type will lead to problems later on.